### PR TITLE
Add install script release notes

### PR DESCRIPTION
**Pull request by ghost**
### What does this PR do?

Add separate release notes and changelog for the install script

### Motivation

Allow users to see the install script changes for each release

### Additional Notes

n/a

### Describe your test plan

Run "inv release.update-install-changelog %versio_number%"

### CODE REVIEW

File: tasks/release.py [hunk 1]
```diff
@@ -102,22 +101,44 @@ def update_changelog(ctx, new_version):
...
+def update_changelog(ctx, new_version):
+    """
+    Quick task to generate the new CHANGELOG using reno when releasing a minor
+    version (linux/macOS only).
+    """
+    _update_changelog(ctx, new_version, "releasenotes", "CHANGELOG")
+
+
+@task
+def update_install_changelog(ctx, new_version):
```
Comment (mx-psi) [inline tasks/release.py:134]: How should this task be run? I am having trouble understanding how we would go about it.
If we were to create the changelog, for, say, 1.0.4, what commands would one have to run?

Comment (mikezhu-dd) [reply tasks/release.py:134]: Similar to update_changelog:
`inv release.update-install-changelog 1.0.4`

I will update https://github.com/DataDog/mars-jenkins-scripts/blob/master/release_documentation/final_artifacts.md#update-the-changelog-release-coordinator with the info.

Comment (albertvaka) [reply tasks/release.py:134]: Similar to `update_changelog`:
`inv release.update-install-changelog 1.0.4`

Comment (mx-psi) [reply tasks/release.py:134]: > Similar to `update_changelog`:
> `inv release.update-install-changelog 1.0.4`

That doesn't work for me though, you need to have a tag (I assume `1.0.4`) and if you do the task fails:

```
1.0.4
fatal: ambiguous argument '1.0.0-devel...remotes/origin/1.-1.x': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
loaded configuration file ./releasenotes/config.yaml
ignoring unknown configuration value 'no_show_source' = True
scanning ./releasenotes/notes (branch=*current* earliest_version=1.0.0-devel collapse_pre_releases=True)
Traceback (most recent call last):
  File "/usr/local/bin/reno", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/site-packages/reno/main.py", line 203, in main
    return args.func(args, conf)
  File "/usr/local/lib/python3.8/site-packages/reno/report.py", line 21, in report_cmd
    ldr = loader.Loader(conf)
  File "/usr/local/lib/python3.8/site-packages/reno/loader.py", line 61, in __init__
    self._load_data()
  File "/usr/local/lib/python3.8/site-packages/reno/loader.py", line 82, in _load_data
    self._scanner_output = self._scanner.get_notes_by_version()
  File "/usr/local/lib/python3.8/site-packages/reno/scanner.py", line 954, in get_notes_by_version
    raise ValueError(
ValueError: earliest-version set to unknown revision '1.0.0-devel'
```

Comment (mikezhu-dd) [reply tasks/release.py:134]: Can you try 
`inv release.update-install-changelog 7.24..0`

The version has to be a git label in the repro.

Comment (mx-psi) [reply tasks/release.py:134]: Oh, so the tag must be on Github, sorry. Okay, one last question. This
https://github.com/DataDog/datadog-agent/blob/de7193d78dfd15f9feedfe96ed19f6bbddcc7a7b/releasenotes-install/config.yaml#L4
makes me think that the install script releases would be tied to the Agent releases, right? Is that what we want? If so this looks good to me but I am not sure about that.

Comment (mikezhu-dd) [reply tasks/release.py:134]: There's some additional work needed. I'll update the PR.

Comment (mikezhu-dd) [reply tasks/release.py:134]: There are even more stuff going on, closing this PR for now.

**Pull request closed without merging**